src/vcdiff.cc

```diff
@@ -177,21 +178,20 @@ void VcdCtx::Process(const char* data, size_t len, bool is_last) {
 
 }
 
-bool VcdCtx::CheckError() {
+bool VcdCtx::CheckError(v8::Isolate* isolate) {
   if (!HasError())
     return true;
 
-  SendError();
+  SendError(isolate);
   return false;
 }
 
-void VcdCtx::SendError() {
-  v8::HandleScope scope;
+void VcdCtx::SendError(v8::Isolate* isolate) {
   v8::Local<v8::Value> args[2] = {
-    v8::String::New(GetErrorString(err_)),
-    v8::Number::New(static_cast<int>(err_)),
+    v8::String::NewFromUtf8(isolate, GetErrorString(err_)),
```

Comment:
I'm no expert also:)
but in nodejs zlib binding seems like they use HandleScope in the same situation:
https://github.com/nodejs/node/blob/master/src/node_zlib.cc#L401